generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -57,10 +57,10 @@ public ValidationResult validateCommandLinePostProfile(Profile profile) {
     private void checkGenerateOutputTarget(ArrayList<String> errorMessages,
                                            OutputTarget outputTarget) {
         if (outputTarget instanceof FileOutputTarget) {
```

Comment:
You could exit early if `outputTarget` isn't a `FileOutputTarget` which could save some of the repeated hard-casts

Reply:
I would rather not introduce changes here as this is being worked on in another ticket (not sure which at the moment)